modules/js/index.js

```diff
@@ -0,0 +1,175 @@
+import { printErrorMsg, addBtn } from './utils.js';
+
+let books = JSON.parse(localStorage.getItem('books'));
+// const addBtn = document.querySelector(".add-btn");
+// const bookList = document.querySelector('#book-list');
+
+// const printErrorMsg = (message) => {
+//   const errMsg = document.querySelector(".err-msg");
+//   errMsg.style.color = "red";
+//   document.querySelector(".err-msg").textContent = message;
+//   setTimeout(() => {
+//     document.querySelector(".err-msg").textContent = "";
+//   }, 2000);
+// };
```

Comment:
- [ ] [OPTIONAL] Because the previous CR didn't mention this it is not a requirement, however, I strongly advise you to remove this piece of commented out code, it will make your code look a lot more professional and readable.